src/services/gapi.js

```diff
@@ -0,0 +1,36 @@
+/* global gapi */
```

Comment:
Probably better to just use `window.gapi` below where we are referencing the global.

Reply:
updated